AttributeError: 'ProxifyHostFile' object has no attribute 'evict'

After changes introduced by https://github.com/dask/distributed/pull/5543, `dask_cuda/tests/test_proxify_host_file.py::test_worker_force_spill_to_disk` is now failing:

```python
___________________________________________ test_worker_force_spill_to_disk ____________________________________________

args = (), kwargs = {}, loop = <tornado.platform.asyncio.AsyncIOLoop object at 0x7f7314ba0580>
injected_func = functools.partial(<function test_worker_force_spill_to_disk at 0x7f7314b723a0>)
cor = functools.partial(<function test_worker_force_spill_to_disk at 0x7f7314b723a0>)

    def test_func(*args, **kwargs):
        with clean() as loop:
            injected_func = functools.partial(func, *args, **kwargs)
            if iscoroutinefunction(func):
                cor = injected_func
            else:
                cor = gen.coroutine(injected_func)

>           loop.run_sync(cor, timeout=timeout)

../distributed/distributed/utils_test.py:808:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../miniconda3/envs/rn-220217/lib/python3.8/site-packages/tornado/ioloop.py:530: in run_sync
    return future_cell[0].result()
dask_cuda/tests/test_proxify_host_file.py:404: in test_worker_force_spill_to_disk
    await client.submit(f)
../distributed/distributed/client.py:287: in _result
    raise exc.with_traceback(tb)
dask_cuda/tests/test_proxify_host_file.py:399: in f
    await w.memory_monitor()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   weight = self.data.evict()
E   AttributeError: 'ProxifyHostFile' object has no attribute 'evict'

../distributed/distributed/worker.py:3752: AttributeError
```

@shwina do you have the time and will to look at this? Otherwise I'll do so.

--- a/dask_cuda/proxify_host_file.py
+++ b/dask_cuda/proxify_host_file.py
@@ -586,7 +586,7 @@ class ProxifyHostFile(MutableMapping):
     def evict(self) -> int:
         """Manually evict 1% of host limit.
 
-        Dask use this to trigger CPU-to-Disk spilling. We don't know how much
+        Dask uses this to trigger CPU-to-Disk spilling. We don't know how much
         we need to spill but Dask will call `evict()` repeatedly until enough
         is spilled. We ask for 1% each time.
 
@@ -608,7 +608,7 @@ class ProxifyHostFile(MutableMapping):
     def fast(self):
         """Alternative access to `.evict()` used by Dask
 
-        Dask expect `.fast.evict()` to be availabe for manually triggering
+        Dask expects `.fast.evict()` to be availabe for manually triggering
         of CPU-to-Disk spilling.
         """
         if len(self.manager._host) == 0:

--- a/dask_cuda/proxify_host_file.py
+++ b/dask_cuda/proxify_host_file.py
@@ -583,25 +583,44 @@ class ProxifyHostFile(MutableMapping):
                 mr = rmm.mr.FailureCallbackResourceAdaptor(current_mr, oom)
                 rmm.mr.set_current_device_resource(mr)
 
+    def evict(self) -> int:
+        """Manually evict 1% of host limit.
+
+        Dask use this to trigger CPU-to-Disk spilling. We don't know how much
+        we need to spill but Dask will call `evict()` repeatedly until enough
+        is spilled. We ask for 1% each time.
+
+        Return
+        ------
+        nbytes: int
+            Number of bytes spilled or -1 if nothing to spill.
+        """
+
+        ret = self.manager.evict(
+            nbytes=int(self.manager._host_memory_limit * 0.01),
+            proxies_access=self.manager._host.buffer_info,
+            serializer=ProxifyHostFile.serialize_proxy_to_disk_inplace,
+        )
+        gc.collect()
+        return ret if ret > 0 else -1
+
     @property
     def fast(self):
-        """Dask use this to trigger CPU-to-Disk spilling"""
+        """Alternative access to `.evict()` used by Dask
+
+        Dask expect `.fast.evict()` to be availabe for manually triggering
+        of CPU-to-Disk spilling.
+        """
         if len(self.manager._host) == 0:
             return False  # We have nothing in host memory to spill
 
         class EvictDummy:
             @staticmethod
             def evict():
-                # We don't know how much we need to spill but Dask will call evict()
-                # repeatedly until enough is spilled. We ask for 1% each time.
                 ret = (
                     None,
                     None,
-                    self.manager.evict(
-                        nbytes=int(self.manager._host_memory_limit * 0.01),
-                        proxies_access=self.manager._host.buffer_info,
-                        serializer=ProxifyHostFile.serialize_proxy_to_disk_inplace,
-                    ),
+                    self.evict(),
                 )
                 gc.collect()
                 return ret

